service/controller/app/v1/resource/cordonchart/create.go

```diff
@@ -0,0 +1,74 @@
+package cordonchart
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/operatorkit/controller/context/resourcecanceledcontext"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/giantswarm/app-operator/pkg/annotation"
+	"github.com/giantswarm/app-operator/service/controller/app/v1/controllercontext"
+	"github.com/giantswarm/app-operator/service/controller/app/v1/key"
+)
+
+type mergeSpec struct {
+	Op    string            `json:"op"`
+	Path  string            `json:"path"`
+	Value map[string]string `json:"value"`
+}
+
+func (r Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	cr, err := key.ToCustomResource(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	if !key.IsCordoned(cr) {
```

Comment:
Early return if app is not cordoned.

Reply:
I figured out I should remove cordon annotations in chart, should not return in here.